### PR TITLE
docs: add MarkText to Vega-Lite ecosystem

### PR DESCRIPTION
**Pull request by fish-404**
Add [MarkText](https://github.com/marktext/marktext) to Vega-Lite ecosystem, a markdown editor support Vega-Lite to generate diagram.

### CODE REVIEW

File: site/ecosystem.md [hunk 1]
```diff
@@ -101,6 +101,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Saite](https://github.com/jsa-aerial/saite) Interactive exploratory graphics and ad hoc visualization application for Clojure(Script). Built on top of [Hanami](https://github.com/jsa-aerial/hanami).
 - [Datablocks](https://datablocks.pro), a node-based editor for exploring, analyzing and transforming data without code.
 - [Rath](https://github.com/Kanaries/Rath) An augmented analysis tool including auto-EDA, pattern discovery, multi-dimensional visualization recommendation, and interactive dashboards generation.
+- [MarkText](https://github.com/marktext/marktext): A simple and elegant open-source markdown editor that focused on speed and usability.
```
Comment (domoritz) [inline site/ecosystem.md:104]:
```suggestion
- [MarkText](https://github.com/marktext/marktext): A markdown editor that focuses on speed and usability.
```

**Review by domoritz — CHANGES_REQUESTED** [review]
Can you explain what you mean by focusing on usability?

File: site/ecosystem.md (hunk 1 above)
Comment (fish-404) [reply site/ecosystem.md:104]: Oh, that just referenced from MarkText description. 
I just an user not the developer. 
I think we can simply use markdown editor as the description if we need more objective.

File: site/ecosystem.md [hunk 2]
```diff
@@ -101,6 +101,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Saite](https://github.com/jsa-aerial/saite) Interactive exploratory graphics and ad hoc visualization application for Clojure(Script). Built on top of [Hanami](https://github.com/jsa-aerial/hanami).
 - [Datablocks](https://datablocks.pro), a node-based editor for exploring, analyzing and transforming data without code.
 - [Rath](https://github.com/Kanaries/Rath) An augmented analysis tool including auto-EDA, pattern discovery, multi-dimensional visualization recommendation, and interactive dashboards generation.
+- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports vega-lite diagram.
```
Comment (domoritz) [inline site/ecosystem.md:104]:
```suggestion
- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite. 
```

File: site/ecosystem.md [hunk 3]
```diff
@@ -101,6 +101,7 @@ We mark featured plugins and tools with a <span class="octicon octicon-star"></s
 - [Saite](https://github.com/jsa-aerial/saite) Interactive exploratory graphics and ad hoc visualization application for Clojure(Script). Built on top of [Hanami](https://github.com/jsa-aerial/hanami).
 - [Datablocks](https://datablocks.pro), a node-based editor for exploring, analyzing and transforming data without code.
 - [Rath](https://github.com/Kanaries/Rath) An augmented analysis tool including auto-EDA, pattern discovery, multi-dimensional visualization recommendation, and interactive dashboards generation.
+- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite.
```
Comment (domoritz) [inline site/ecosystem.md:104]:
```suggestion
- [MarkText](https://github.com/marktext/marktext): An open-source markdown editor that supports Vega-Lite.
```

**Pull request merged**